go/store/types/apply_map_edits.go

```diff
@@ -36,7 +37,7 @@ type EmptyEditProvider struct{}
 
 // Next will always return nil
 func (eep EmptyEditProvider) Next() (*KVP, error) {
-	return nil, nil
```

Comment:
Something I changed for every EditProvider implementation is that it now returns io.EOF when all items have been read.